darts/tests/models/forecasting/test_historical_forecasts.py

```diff
@@ -0,0 +1,735 @@
+import unittest
+
+import numpy as np
+
+from darts import TimeSeries
+from darts.dataprocessing.transformers import Scaler
+from darts.datasets import AirPassengersDataset
+from darts.logging import get_logger
+from darts.models import (
+    BlockRNNModel,
+    CatBoostModel,
+    LightGBMModel,
+    LinearRegressionModel,
+    NBEATSModel,
+    RNNModel,
+    TCNModel,
+    TFTModel,
+    TransformerModel,
+)
+from darts.tests.base_test_class import DartsBaseTestClass
+from darts.utils import timeseries_generation as tg
+from darts.utils.likelihood_models import GaussianLikelihood
+
+try:
+    import torch
+
+    TORCH_AVAILABLE = True
+except ImportError:
+    logger = get_logger(__name__)
+    logger.warning("Torch not installed - will be skipping Torch models tests")
```

Comment:
```suggestion
    logger.warning("Torch not installed - will be skipping historical forecasts tests")
```
Could we maybe simply skip torch models from the tests but keep the other tests in case torch is not available?